code/modules/projectiles/guncrafting/attachments/energy_bullets.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/weapon/gun_attachment/energy_bullet
+	gun_type = CUSTOMIZABLE_ENERGY
+	uses_overlay = FALSE
+	icon_state = "bullet"
+	not_okay = /obj/item/weapon/gun_attachment/energy_bullet
+
+/obj/item/weapon/gun_attachment/energy_bullet/disorienter
+	name = "Energy Disorienter"
+	desc = "Drowsiness, less damage."
+
+/obj/item/weapon/gun_attachment/energy_bullet/disorienter/on_fire(var/obj/item/weapon/gun/owner, var/obj/item/projectile/bullet)
+	..()
+	bullet.drowsy += 10
+	if(bullet.damage >= 5)
+		bullet.damage -= 5
+	else
+		bullet.damage = 0
+
+/obj/item/weapon/gun_attachment/energy_bullet/decloner
```

Comment:
Ah, the Decloner mod. I don't know why goof put it in as a energy bullet modification. This would serve better as being its own gun base instead of a modification.